data/crates.json

```diff
@@ -958,6 +958,17 @@
                                 "name": "notify",
                                 "notes": "Watch files or directories and execute a function when they change"
                             }]
+                        },
+                        {
+                            "name": "User directories",
+                            "recommendations": [{
+                                "name": "dirs",
+                                "notes": "Provide platform-specific locations for configuration, cache, and other data"
+                            }],
+			    "see_also": [{
+				"name": "directories",
+				"notes": "A higher-level library that can also compute paths for applications"
+			    }]
```

Comment:
The indentation looks a bit iffy here, would you mind cleaning that up?

Reply:
Sorry, damn tabs. Force-pushed a fix.